tests/Ruby30Test.php

```diff
@@ -17,6 +17,9 @@ class Ruby30Test extends Base
         ...Base::BAR_RESPONSES,
         ...Base::GENERAL_RESPONSES,
         ...Base::LARGE_FILE_RESPONSES,
```

Comment:
I see four file upload requests but here I see 5 upload outputs, (one from GENERAL_RESPONSES and 4 from LARGE_FILE_RESPONSES